src/content/contribute/index.md

```diff
@@ -45,7 +45,6 @@ The remainder of this section of the site is dedicated to developers such as you
 
 - [Writing a Loader](/contribute/writing-a-loader)
 - [Writing a Plugin](/contribute/writing-a-plugin)
-- [Writing a Scaffold](/contribute/writing-a-scaffold)
```

Comment:
Let's add a redirection for this in `webpack.ssg.js`, as there might be links somewhere to it, e.g.:

```js
      new RedirectWebpackPlugin({
        redirects: {
         '/contribute/writing-a-scaffold': '/api/cli/#init',
        }
    })
```
Same applied to the removed `src/content/guides/scaffolding.md`.

Reply:
Did the same and fixed other grammatical errors. Thanks.